### DISCUSSION TITLE
2.x文档无法访问了

### DISCUSSION THREAD

**Discussion by xch1029** [ant-design/ant-design-mobile#6222] [Q&A]
https://antd-mobile-v2.surge.sh/index-cn

**Comment by zombieJ** [comment]
我这边可以打开，看看是不是要开个代理什么的

**Reply by xch1029** [reply]
好吧，是挺奇怪的
最新的文档是正常的 https://mobile.ant.design/zh
2.x的一直加载中 https://antd-mobile-v2.surge.sh/index-cn

**Reply by weiwentao518** [reply]
我也打不开

**Comment by miracles1919** [comment]
大概是网络问题

![image](https://github.com/ant-design/ant-design-mobile/assets/22469543/eae7ab90-30b0-4125-8840-24a4874546b1)

**Reply by xch1029** [reply]
国内有镜像站点吗

**Reply by miracles1919** [reply]
v5 有, v2 不太清楚..

**Reply by sphenginx** [reply]
我这边ping没问题，但是打开后是个白屏页面……

**Comment by sphenginx** [comment]
我这边ping没问题：
![image](https://github.com/ant-design/ant-design-mobile/assets/1829395/6f85edb6-ce50-4b24-896f-96a5036bea8a)


但是打开页面之后是白屏：
![image](https://github.com/ant-design/ant-design-mobile/assets/1829395/be9a8524-344b-48fe-bb78-ca78b2fa3994)

**Comment by weiwentao518** [comment]
![image](https://github.com/user-attachments/assets/98dfbc68-3b33-4459-b90c-720b1249910b)
同样打不开，能解决一下吗？？？
